wrappers/javascript/packages/anoncreds-shared/src/api/Credential.ts

```diff
@@ -140,4 +148,32 @@ export class Credential extends AnoncredsObject {
     const index = anoncreds.credentialGetAttribute({ objectHandle: this.handle, name: 'rev_reg_index' })
     return index ? Number(index) : undefined
   }
+
+  public toW3C(options: CredentialToW3COptions): W3CCredential {
+    let credential
+    // Objects created within this method must be freed up
+    const objectHandles: ObjectHandle[] = []
+    try {
+      const credentialDefinition =
+        options.credentialDefinition instanceof CredentialDefinition
+          ? options.credentialDefinition.handle
+          : pushToArray(CredentialDefinition.fromJson(options.credentialDefinition).handle, objectHandles)
+
+      credential = new W3CCredential(
+        anoncreds.credentialToW3C({
+          objectHandle: this.handle,
+          credentialDefinition
+        }).handle
+      )
+    } finally {
+      objectHandles.forEach((handle) => {
+        handle.clear()
+      })
+    }
+    return credential
+  }
+
+  public static fromW3C(options: CredentialFromW3COptions) {
```

Comment:
Won't comment again, but would like to use `w3c` or `W3c` (depending on whether it's the start of the variable /method name), but not use `W3C` (all caps)